constellation/src/storage/mem_store.rs

```diff
@@ -226,13 +229,15 @@ impl LinkReader for MemStorage {
                 version: (0, 0),
                 items: Vec::new(),
                 next: None,
+                total: 0,
             });
         };
         let Some(did_rkeys) = paths.get(&Source::new(collection, path)) else {
             return Ok(PagedAppendingCollection {
                 version: (0, 0),
                 items: Vec::new(),
                 next: None,
+                total: 0,
             });
```

Comment:
not for this PR but i wish i knew about `#[derive(Default)]` before i wrote some of this!